packages/DApp/.mocharc.json

```diff
@@ -1,11 +1,13 @@
 {
   "spec": "test/**/*.test.{ts,tsx}",
-  "require": ["ts-node/register", "jsdom-global/register"],
-  "watchExtensions": "ts",
+  "require": [
+    "jsdom-global/register"
+  ],
   "extension": "ts",
   "timeout": 3000,
   "node-option": [
     "experimental-specifier-resolution=node",
     "loader=ts-node/esm"
-  ]
+  ],
+  "exit": true
```

Comment:
Similarly to https://github.com/waku-org/js-waku/issues/766, some handles are probably left open and hang. Most likely `js-waku` client since the hook here wrapping it doesn't have a clean-up fn and `client.stop()` ins't called. 

Mocking the client, could be another approach.